### PR TITLE
Opgops 1691

### PR DESCRIPTION
**Pull request by brettminnie**
Add container for dashboards to monitoring server config

### CODE REVIEW

File: elastic-beats/templates/metricbeat.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        - memory
+        - network
+        - process
+        - socket
+      enabled: true
+      period: 10s
+      processes: ['.*']
+
+    #------------------------------- Docker Module -------------------------------
+    #- module: docker
```
Comment (tim-mcarthur-moj) [inline elastic-beats/templates/metricbeat.yml.j2]: is this not going to replace the dockbeats.  so should be enabled in the template

**Review by tim-mcarthur-moj — CHANGES_REQUESTED** [review]

**Comment by brettminnie** [comment]
:+1:

Just need to test it before I do that

File: elastic-beats/templates/topbeat.yml.j2 [hunk 2]
```diff
@@ -1,165 +0,0 @@
-################### Topbeat Configuration Example #########################
-
-############################# Input ############################################
-input:
-  period: 10
-  procs: ["php.*", "nginx.*","java.*","docker.*","mongo.*","salt.*","python.*","collectdmon.*"]
```
Comment (tim-mcarthur-moj) [inline elastic-beats/templates/topbeat.yml.j2:6]: this config needs to be transferred to metricbeats.  this reduces the disk space requirement and is targeted for our applications. this should probably be set based on the elastic-beats:metricbeat pillar

**Review by tim-mcarthur-moj — CHANGES_REQUESTED** [review]

**Comment by brettminnie** [comment]
:+1:

It's of course named processes in metricbeats

File: monitoring/templates/compose-monitoring-server.yml [hunk 3]
```diff
@@ -43,25 +39,22 @@ elasticsearch:
...
-  environment:
-    SHARED_DATA_BASE: '/tmp'
-    SHARED_DATA_PATHS: "['/elasticsearchshareddata/beats']"
-    DASHBOARD_SLEEP_TIMEOUT: 90
   volumes:
+{% if remote_elastic == 'no' -%}
+  # data on volume
     - {{ monitoring.server.elasticsearch.data_dir }}:/usr/share/elasticsearch/data
-
 {%- else %}
```
Comment (tim-mcarthur-moj) [inline monitoring/templates/compose-monitoring-server.yml]: how do the templates get imported when **remote_elastic** is 'no'

**Review by tim-mcarthur-moj — CHANGES_REQUESTED** [review]

**Review by tim-mcarthur-moj — APPROVED** [review]

**Pull request merged**